app/models/contract.rb

```diff
@@ -21,7 +21,7 @@ class Contract < ApplicationRecord
   has_many :issues
   has_many :risks
   has_many :lessons
-  # has_many :notes, as: :noteable, dependent: :destroy
+  has_many :notes, as: :noteable, dependent: :destroy
```

Comment:
This is conflicting with the field we have in database for contract. Please comment this line to prevent any error.